students/Vincent_Aquila/session03/slicing_lab.py

```diff
@@ -0,0 +1,65 @@
+"""
+Python210A Vincent Aquila  Fall2018
+using Python 3.7.0
+assignment objective: Slicing Lab
+
+Write some functions that take a sequence as an argument, and return a copy of that sequence:
+
+1) with the first and last items exchanged.
+2) with every other item removed.
+3) with the first 4 and the last 4 items removed, and then every other item in the remaining sequence.
+4) with the elements reversed (just with slicing).
+5) with the last third, then first third, then the middle third in the new order.
+NOTE: These should work with ANY sequence – but you can use strings to test, if you like.
+
+Your functions should look like:
+
+def exchange_first_last(seq):
+    return a_new_sequence
+"""
+
+#item 1
+print("---Item 1: with the first and last items exchanged")
+def swap_first_last(list):
+    x = list[:1]
```

Comment:
this should work for any sequence -- not only lists! and it should return the same type of sequence that got passed in!